test/end-to-end/config.json

```diff
@@ -13,7 +13,7 @@
             "name": "automation",
             "description": "UI automation recipe",
             "modules": [],
-            "packages": [{"name": "xz", "version": "5.1.2"}, {"name": "whois", "version": "5.1.1"}]
+            "packages": [{"name": "httpd", "version": ""}]
```

Comment:
Can you please put a rationale into the commit log? I. e. not the *what*, but the *why*? What's wrong with testing a particular version, and why not test with two packages?

Reply:
+1

Reply:
@martinpitt, @atodorov, thanks for your review. I'll add more descriptions into commit.